sycl/source/detail/plugin.hpp

```diff
@@ -231,6 +231,15 @@ class plugin {
   void *getLibraryHandle() { return MLibraryHandle; }
   int unload() { return RT::unloadPlugin(MLibraryHandle); }
 
+  // Get backend optimization option
+  void getBackendOptimizationOption(int opt_level,
+                                    char **backend_option) const {
+    [[maybe_unused]] auto pi_result =
```

Comment:
does just

`call_nocheck<PiApiKind::piPluginGetBackendOptimizationOption>(
            opt_level, backend_option);` work without assigning it to anything?

Reply:
I think it's a standard way to write plugin APIs such that they return pi_result. It will make it easier to change the API logic if needed.

Thanks